dxr/static/templates/path_line.html

```diff
@@ -4,7 +4,7 @@
 {%- if not is_binary or is_dir -%}
 <a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
 {%- endif -%}
-    {{ display_path }}
+    {{ display_path|safe }}
```

Comment:
Hmm, we can't have that. We shouldn't have to trust committers to every tree to prevent XSS attacks. So I guess we need to either find a way to get rid of |safe here or explicitly escape path segments.